### PR TITLE
PA-1929 Updating Project Risks

### PR DESCRIPTION
**Pull request by Fosol**
Removed the `Completed` project risk and resorted them so that they are now only `Green`, `Yellow` and `Red.

- Added DB migration **v01.05.00**
- Fixed FastSelect value undefined
- Updated keycloak realm config.  We had not updated the config to include **SRES Financial Role** previously

### CODE REVIEW

File: auth/keycloak/config/realm-export.json [hunk 1]
```diff
@@ -851,7 +878,7 @@
         "account": ["view-profile", "manage-account"]
       },
       "notBefore": 0,
-      "groups": ["/Real Estate Manager", "/SRES"]
+      "groups": ["/Real Estate Manager", "/SRES", "/SRES Financial Manager"]
```
Comment (Fosol) [inline auth/keycloak/config/realm-export.json:881]: Give the `sres` user the `SRES Financial Manager` role

File: auth/keycloak/config/realm-export.json [hunk 2]
```diff
@@ -246,6 +246,24 @@
         "clientRole": false,
```
Comment (Fosol) [inline auth/keycloak/config/realm-export.json:246]: We forgot to update the keycloak config.  Didn't notice this until now.

File: frontend/src/components/common/form/FastSelect.tsx [hunk 3]
```diff
@@ -105,7 +105,7 @@ export const FastSelect: React.FC<FastSelectProps> = memo(
         return null;
       }
       options.forEach((x: any) => {
-        if (x.value === value.toString()) {
+        if (x.value === value?.toString()) {
```
Comment (Fosol) [inline frontend/src/components/common/form/FastSelect.tsx:108]: Fix an `undefined` bug

File: frontend/src/features/projects/common/forms/UpdateInfoForm.tsx [hunk 4]
```diff
@@ -61,34 +102,20 @@ const UpdateInfoForm = ({
...
-          </div>
-          <div>
-            <small>{tier3Tooltip}</small>
-          </div>
-          <div>
-            <small>{tier4Tooltip}</small>
-          </div>
-        </Form.Group>
+        <Tooltip>
+          <small>{tierLevelTooltip}</small>
```
Comment (Fosol) [inline frontend/src/features/projects/common/forms/UpdateInfoForm.tsx:117]: Reduce the scrolling for annoying text.

File: frontend/src/features/projects/common/forms/UpdateInfoForm.tsx [hunk 5]
```diff
@@ -99,28 +126,14 @@ const UpdateInfoForm = ({
...
-              </div>
-              <div>
-                <small>{risk3Tooltip}</small>
-              </div>
-              <div>
-                <small>{risk4Tooltip}</small>
-              </div>
-            </Form.Group>
+            <Tooltip>
+              <small>{riskTooltip}</small>
```
Comment (Fosol) [inline frontend/src/features/projects/common/forms/UpdateInfoForm.tsx:135]: Reduce the scrolling for annoying text.

File: frontend/src/features/projects/common/strings.ts [hunk 6]
```diff
@@ -2,18 +2,19 @@ export const clickableTooltip = 'Click to view property details';
...
   'Please make any necessary updates to the valuation information and the classification.';
 
-export const tier1Tooltip = 'Tier 1: Properties with a net value of less than $1M.';
-export const tier2Tooltip =
-  'Tier 2: Properties with a net value of $1M or more and less than $10M.';
-export const tier3Tooltip =
-  'Tier 3: Properties are single parcels with a net value of $10M or more.';
-export const tier4Tooltip =
-  'Tier 4: Properties are multiple parcels with a cumulative net value of $10M or more.';
+export const tierTooltips = {
```
Comment (Fosol) [inline frontend/src/features/projects/common/strings.ts:5]: Group related consts.

File: frontend/src/features/projects/common/strings.ts [hunk 7]
```diff
@@ -2,18 +2,19 @@ export const clickableTooltip = 'Click to view property details';
...
+  tier1Tooltip: 'Tier 1: Properties with a net value of less than $1M.',
```
Comment (goemen) [inline frontend/src/features/projects/common/strings.ts:6]: Change to `Properties with a net value less than $1M.`

**Review by goemen — APPROVED** [review]
Looks good to me

**Review by jdtoombs — APPROVED** [review]
Less scrolling the better 👍

**Pull request merged**